Dockerfile

```diff
@@ -0,0 +1,14 @@
+ARG GOVERSION=latest
+FROM golang:$GOVERSION AS builder
+
+WORKDIR /src
+COPY . .
+
+RUN GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -o . -ldflags="-s -w -X main.release=$(git rev-parse HEAD)" .
```

Comment:
`main.release=` assumes having "release" variable in the "main" package, so we should either add that, or drop this (since it'll be unused).